cmd/cephfs/main.go

```diff
@@ -31,6 +31,7 @@ var (
 	nodeID          = flag.String("nodeid", "", "node id")
 	volumeMounter   = flag.String("volumemounter", "", "default volume mounter (possible options are 'kernel', 'fuse')")
 	metadataStorage = flag.String("metadatastorage", "", "metadata persistence method [node|k8s_configmap]")
+	mountCacheDir   = flag.String("mountcachedir", "", "mount info cache save dir")
```

Comment:
need deployment document updation for this new variable.

do we need to update our nodeplugin daemonset for this one?

Reply:
question, if this is a host directory or local to the container. if host directory how it behaves for multiple cephfs-drivers?

and will this work for both metadataStorage and nodecache storage?

Reply:
> question, if this is a host directory or local to the container. if host directory how it behaves for multiple cephfs-drivers?
> 
> and will this work for both metadataStorage and nodecache storage?

multiple cephfs-drivers  can share  metadataStorage,  it just check if  volume is exist when load cache info from local  cache dir. 

volume mount cache support user defined, if   multiple cephfs-drivers user should different host cache dir.   do we need suport if user define same  local dir when node has multiple cephfs-drivers ?